src/client/Microsoft.Identity.Client/AppConfig/BaseAbstractApplicationBuilder.cs

```diff
@@ -218,13 +217,6 @@ internal void ResolveAuthority()
                 // Both WithAuthority and WithTenant were used at app config level
                 if (!string.IsNullOrEmpty(Config.TenantId))
```

Comment:
So the new behavior is ignoring the tenant for ADFS authorities?

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/5daef991982e092d2eeb4b6047cf8de585d5e5cd/src/client/Microsoft.Identity.Client/Instance/AdfsAuthority.cs#L58-L61

Reply:
Yes.

Reply:
ADFS team (Bryant) confirm that even if there is a `tid` claim in the ADFS token, ADFS is not tenanted and it should be ignored.